src/gui/guiFormSpecMenu.h

```diff
@@ -27,6 +27,7 @@ with this program; if not, write to the Free Software Foundation, Inc.,
 #include "inventorymanager.h"
 #include "modalMenu.h"
 #include "guiInventoryList.h"
+#include "guiScrollBar.h"
```

Comment:
Not needed in the header. Please move to `guiFormSpecMenu.cpp`

Reply:
Two things: a) it's already included in `guiTable.h` and b)

https://github.com/minetest/minetest/blob/master/src/gui/guiFormSpecMenu.h#L375
```
GUIScrollBar::ArrowVisibility arrow_visiblity = GUIScrollBar::DEFAULT;
```

will not work with a forwarded class declaration.  The only reason why that worked in the scrollbaroptions PR was because `guiTable.h` included it.

Reply:
Thanks. Didn't see that.